recipes/libtlx/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "libtlx" %}
+{% set version = "0.5.20191212" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "5e67d3042a390dbb831b6d46437e3c7fadf738bff362aa7376b210b10ecd532d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - url: 'https://github.com/tlx/tlx/archive/v{{version}}.tar.gz'
+    {{hash_type}}: {{hash_value}}
```

If values are only used one I prefer not to use a variable:
```suggestion
    sha256: 5e67d3042a390dbb831b6d46437e3c7fadf738bff362aa7376b210b10ecd532d
```